docs/platform.rst

```diff
@@ -737,6 +737,20 @@ In order to use QT, import it from Sgtk::
 Toolkit will make sure Qt is sourced in the correct way. Keep in mind that many applications (for example Nuke)
 may not have a functional Qt that can be imported when they run in batch mode.
 
+When running in an environment with PySide 2, Toolkit will provide under ``sgtk.platform.qt`` an emulation
```

Comment:
I think this paragraph should have its own subheading in the docs so it shows up in the toc tree!